.travis.yml

```diff
@@ -1,37 +1,60 @@
 language: cpp
+#services:
+#  - docker
+
+arch:
+  - amd64
+  - arm64
 
 os:
   - linux
   - osx
+  - windows
 
 compiler:
   - gcc
 
+matrix:
+  fast_finish: true
+  include:
+    - os: linux
+      arch: amd64
+    - os: linux
+      arch: arm64
+    - os: windows
+    - os: osx
+
 addons:
   apt:
     packages:
     - libgtk-3-dev
     - nasm
+    - build-essential
+    - cmake
+    - libjpeg-dev
 
 before_script:
+#  - docker build --tag openpnp-capture-rpi3-arm64:latest . -f docker/Dockerfile.rpi3_arm64 
   - mkdir build
   - cd build
   - cmake ..
 
 script: make
+# docker run -v $PWD:/openpnp-capture openpnp-capture-rpi3-arm64 make
 
 before_deploy:
   - mkdir deploy
+  - if [ $TRAVIS_OS_NAME = windows ]; then cp libopenpnp-capture.dll deploy; cp linux/tests/openpnp-capture-test deploy/openpnp-capture-test-windows; fi
   - if [ $TRAVIS_OS_NAME = osx ]; then cp libopenpnp-capture.dylib deploy; cp mac/tests/openpnp-capture-test deploy/openpnp-capture-test-mac; fi
   - if [ $TRAVIS_OS_NAME = linux ]; then cp libopenpnp-capture.so deploy; cp linux/tests/openpnp-capture-test deploy/openpnp-capture-test-linux; fi
+  - if [ $TRAVIS_CPU_ARCH = arm64 ]; then cp libopenpnp-capture.so deploy; cp linux/tests/openpnp-capture-test deploy/openpnp-capture-test-linux-arm64; fi
```

Comment:
I think this is going to overwrite the Linux x64 build, since it has the same name. I'm going to test some changes to append the platform and arch to each binary to make everything clearer.

Reply:
IIRC concurrent job builds are executed in separate contexts/VMs/filesystems? The x86 build should not affect the arm64 one... I think, I might be wrong though :-S

Reply:
Ohh, I see what you mean now, overwritten on deploy time, of course! :-/

Reply:
Do you have the changes you refer to (append platform) in some other branch or staged locally? Let's ship this PR and refine the appveyor vs travis windows migration things on another PR? I want to get the RPi3 functionality shipped first ;)

Reply:
No - I started on it then got wrapped up in Windows stuff. If you just want to change it so that it won't overwrite on deploy we can go from there.